### ISSUE TITLE
Can't install: psr/log (locked at 1.0.2)

### ISSUE THREAD

**Issue by ultrablue** [Adldap2/Adldap2-Laravel#726]
- Laravel Version: 5.6
- Adldap2-Laravel Version: Current
- PHP Version: 7.2
- LDAP Type: NA

### Description:
When I try to install adldap2-laravel, Composer refuses, saying:

```
$ composer require adldap2/adldap2-laravel
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Using version ^6.0 for adldap2/adldap2-laravel
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install adldap2/adldap2-laravel v6.0.5
    - Conclusion: don't install adldap2/adldap2-laravel v6.0.4
    - Conclusion: don't install adldap2/adldap2-laravel v6.0.3
    - Conclusion: don't install adldap2/adldap2-laravel v6.0.2
    - Conclusion: don't install adldap2/adldap2-laravel v6.0.1
    - Conclusion: remove psr/log 1.0.2
    - Installation request for adldap2/adldap2-laravel ^6.0 -> satisfiable by adldap2/adldap2-laravel[v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4, v6.0.5].
    - Conclusion: don't install psr/log 1.0.2
    - adldap2/adldap2-laravel v6.0.0 requires adldap2/adldap2 ^10.0 -> satisfiable by adldap2/adldap2[v10.0.0, v10.0.1, v10.0.2, v10.0.3, v10.0.4, v10.0.5, v10.0.6, v10.0.7, v10.0.8, v10.0.9].
    - adldap2/adldap2 v10.0.0 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.1 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.2 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.3 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.4 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.5 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.6 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.7 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.8 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - adldap2/adldap2 v10.0.9 requires psr/log ~1.1 -> satisfiable by psr/log[1.1.0, 1.1.x-dev].
    - Can only install one of: psr/log[1.1.0, 1.0.2].
    - Can only install one of: psr/log[1.1.x-dev, 1.0.2].
    - Installation request for psr/log (locked at 1.0.2) -> satisfiable by psr/log[1.0.2].


Installation failed, reverting ./composer.json to its original content.
```
Any ideas about how I can get going?

Thanks!
Blue

**Comment by stevebauman** [comment]
Hey @ultrablue,

Whoops! I locked Adldap2 v6 to version ~1.1. I'm going to update it now to use ~1.0.

**Comment by ultrablue** [comment]
Awesome, thanks so much!

**Comment by stevebauman** [comment]
No problem! :smile:

Released [v10.0.10](https://github.com/Adldap2/Adldap2/releases/tag/v10.0.10), run `composer update` and you'll be all set!

**Comment by ultrablue** [comment]
👍🏼 Thanks again!
